SCClassLibrary/Common/UnitTesting/UnitTest.sc

```diff
@@ -33,24 +37,41 @@ UnitTest {
 		if(method.isNil) {
 			Error("Test method not found " + methodName).throw
 		};
-		class.new.runTestMethod(method);
+		this.use {
+			class.new.runTestMethod(method)
+		}
 	}
 
+	// this should be called when a new unit test instance is created
+	*setUpClass {}
+
+	// this should be called when a unit test instance is abandoned
+	*tearDownClass {}
```

Comment:
these comments make it sound as if the functions should be called by test code; something like "called before running tests in a unit test class" would be more appropriate.

Reply:
yes, done.